skills/unraid/references/troubleshooting.md

```
@@ -1,36 +1,109 @@
+
+**Symptoms:** Timeout, connection refused, network error
+
+**Diagnostic steps:**
+
+1. Test basic connectivity:
+
+```python
+unraid(action="health", subaction="test_connection")
+```
+
+2. Full diagnostic report:
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)